### PR TITLE
runtime context: unregister stores of disposed arcs and add tests

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

File: src/runtime/tests/runtime-test.ts [hunk 1]
```diff
@@ -61,4 +62,44 @@ describe('Runtime', () => {
     const actual = await Runtime.loadManifest('./src/runtime/tests/artifacts/test.manifest', loader, registry);
     assertManifestsEqual(actual, expected);
   });
+  it.only('runs arcs', async () => {
```
Comment (csilvestrini) [inline src/runtime/tests/runtime-test.ts]: it.only -> it

File: src/runtime/tests/runtime-test.ts [hunk 2]
```diff
@@ -61,4 +62,44 @@ describe('Runtime', () => {
...
+    assert.equal(runtime.arcById.size, 0);
+    const arc = runtime.runArc('test-arc', 'volatile://');
+    assert.isNotNull(arc);
+    assert.equal(runtime.arcById.size, 1);
+    runtime.runArc('test-arc', 'volatile://');
+    assert.equal(runtime.arcById.size, 1);
+    runtime.runArc('other-test-arc', 'volatile://');
+    assert.equal(runtime.arcById.size, 2);
+  });
+  it.only('registers and unregisters stores', async () => {
```
Comment (csilvestrini) [inline src/runtime/tests/runtime-test.ts]: here too

File: src/runtime/tests/runtime-test.ts [hunk 3]
```diff
@@ -61,4 +62,44 @@ describe('Runtime', () => {
     const actual = await Runtime.loadManifest('./src/runtime/tests/artifacts/test.manifest', loader, registry);
     assertManifestsEqual(actual, expected);
   });
+  it.only('runs arcs', async () => {
+    const runtime = Runtime.getRuntime();
+    assert.equal(runtime.arcById.size, 0);
+    const arc = runtime.runArc('test-arc', 'volatile://');
+    assert.isNotNull(arc);
+    assert.equal(runtime.arcById.size, 1);
```
Comment (csilvestrini) [inline src/runtime/tests/runtime-test.ts]: Optional: I think this is a little more explicit: `assert.hasAllKeys(runtime.arcById, ['test-arc'])`

File: src/runtime/tests/runtime-test.ts (hunk 1 above)
Comment (mmandlis) [reply src/runtime/tests/runtime-test.ts]: duh :( fixed

File: src/runtime/tests/runtime-test.ts (hunk 3 above)
Comment (mmandlis) [reply src/runtime/tests/runtime-test.ts]: updated

**Pull request merged**